lib/services/consumers/application.dao.js

```diff
@@ -53,11 +53,10 @@ dao.update = function (id, props) {
     });
 };
 
-dao.findAll = function (query) {
+dao.findAll = function ({ start = 0, count = '100' } = {}) {
```

Comment:
a little strange that `count` is a string. Is it required by the ioredis library? 
What will happen if I pass not a number? No validation as I understand?

Reply:
Good point. I set it as string because Redis is accepting strings only and — given that http query parameters are strings as well, I saw a perfect match. I'll look into, if it accepts numbers as well then better.